src/features/RentWorks/common/PropertyOwnerInfoCard.jsx

```diff
@@ -90,6 +103,9 @@ export default function PropertyOwnerInfoCard({
     }
   }, [property?.id]);
 
+  // TODO : fix this
+  const tenant = {};
```

Comment:
todo comment here. we might need to keep the above rtk queries so that we can have the tenant details.